app/src/main/java/com/github/polypoly/app/network/RemoteDB.kt

```diff
@@ -0,0 +1,128 @@
+package com.github.polypoly.app.network
+
+import com.github.polypoly.app.game.User
+import com.github.polypoly.app.global.Settings.Companion.DB_USERS_PROFILES_PATH
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
+import com.google.firebase.database.ktx.getValue
+import java.util.concurrent.CompletableFuture
+
```

Comment:
Missing documentation (not talking about private and/or overridden functions)

Reply:
Done 👍